### ISSUE TITLE
Can't load file with multiple documents

### ISSUE THREAD

**Issue by lufego** [wwilsman/js-yaml-loader#7]
### The problem

When trying to load a file with multiple documents, I get the following error:

`Module Error (from ./node_modules/js-yaml-loader/index.js):
expected a single document in the stream, but found more`

The file looks like something like this:
````yaml
---
kind: example
apiVersion: v1
name: test
---
kind: example2
apiVersion: v1
name: test2
---
kind: example3
apiVersion: v1
name: test3
````

### The solution
Ideally, I'd add a checker to count the documents, and if length > 1, then I'd invoke the `safeLoadAll` method instead of `safeLoad`. But since `js-yaml` doesn't export this counter function, would it be possible to replace `safeLoad` to `safeLoadAll`? I've tested and it would load both single and multiple documents in a file.

### The change

````js
const uneval = require('un-eval');
const { getOptions } = require('loader-utils');
const yaml = require('js-yaml');

module.exports = function(source) {
  this.cacheable && this.cacheable();

  try {
    const options = getOptions(this) || {};
    const safe = options.safe !== false;
    const res = safe ? yaml.safeLoadAll(source) : yaml.loadAll(source);
    return `module.exports = ${uneval(res)};`;
  } catch (err) {
    this.emitError(err);
    return null;
  }
};

````

**Comment by wwilsman** [comment]
Thanks @lufego, that would be a nice addition! Feel free to open a PR, otherwise I'll add this over the weekend

**Comment by wwilsman** [comment]
Hey @lufego!

I just released a new version (v1.1.0) that should work with multi-document structures and still work with single documents as well!
